SwiftReflector/Enums.cs

```diff
@@ -149,5 +149,22 @@ public enum ReflectionStrategy {
 		Compiler,
 		Parser,
 	}
+
+	public enum TargetCpu {
+		Arm64,
+		Armv7,
+		Armv7s,
+		I386,
+		X86_64,
+	}
+
+	public enum TargetManufacturer {
+		Apple,
+	}
+
+	public enum TargetEnvironment {
+		Device,
+		Simulator,
```

Comment:
It's often called _variant_
and it's also used for Catalyst, being an macOS variant

Right now I'm not quite sure how you would represent this...

Reply:
What's the extended target triple for that?

Reply:
I don't recall... that was not my area of interest back then :)

A quick google suggest `x86_64-apple-ios-macabi` has being used (and that rings a bell, similar to what I used for xtro). Replace first part with `arm64` when needed.

The xcframework store the binaries inside a `ios-arm64_x86_64-maccatalyst` directory, see https://github.com/spouliot/xcframework

Reply:
I think it will just come down to adding a `MacCatalyst` to the TargetEnvironment. It will require a change to the build script, but LA LA LA I CAN'T HEAR YOU